test/e2e/cypress/plugins/index.js

```diff
@@ -15,18 +15,27 @@
 /**
  * @type {Cypress.PluginConfig}
  */
-// eslint-disable-next-line no-unused-vars
 
+const cypressSplit = require('cypress-split');
+const { exec } = require('child_process');
 const http = require('http');
 const webpack = require('@cypress/webpack-preprocessor');
 let heartbeatsIntervals = [];
 
 module.exports = (on, config) => {
   // `on` is used to hook into various events Cypress emits
   // `config` is the resolved Cypress config
+  on('before:run', async () => {
+    const photofinishBinary =
+      await getPhotofinishBinaryAndGiveExecutablePermissions();
+    await runPhotofinishMainScenario(photofinishBinary);
+  });
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏This means the scenario is loaded once per process, right?
So, given the target application is the same, the CI will be executed 8 times while on local execution just once.

Reply:
that's it, once per process, so if you execute the whole test suite locally (sequential) this will be executed just once at the very beginning.